### PR TITLE
edited vendor list

### PR DESCRIPTION
**Pull request by ccsouthard**

### CODE REVIEW

**Comment by ccsouthard** [comment]
Why is Netlify in this repo? I didn't see any config?

**Comment by notslang** [comment]
Netlify is in the repo because I was checking to see if it could build make files, no config is written.

File: view/layout.marko [hunk 1]
```diff
@@ -84,6 +84,9 @@ html
           li
             a href="http://www.technistone.eu/en/" title="Technistone"
               -- Technistone
+           li
```
Comment (notslang) [inline view/layout.marko:87]: this line isn't indented properly

**Comment by notslang** [comment]
fixed in c077e37bc36143245ab87fd2cc3829cca836bbcb

**Pull request closed without merging**